legend-engine-executionPlan-execution/src/main/java/org/finos/legend/engine/plan/execution/result/ErrorResult.java

```diff
@@ -17,6 +17,9 @@
 import org.apache.commons.lang3.exception.ExceptionUtils;
 import org.finos.legend.engine.protocol.pure.v1.model.SourceInformation;
 import org.finos.legend.engine.shared.core.operational.errorManagement.EngineException;
+import org.slf4j.Logger;
+
+import java.sql.Connection;
```

Comment:
ErrorResult being in legend-engine-executionPlan-execution module should ideally not depend on java.sql.Connection (which is a relational concept)

Closing the child result should automatically trigger closing the connection (or) A relational subtype of ErrorResult needs to be introduced

Reply:
I subtyped ErrorResult to a RelationalErrorResult